components/signersForm.tsx

```diff
@@ -250,6 +250,7 @@ const SignersForm: FC<{
                               {index === 0 ? "Owner Name" : ""}
                             </label>
                             <Field
+                              disabled={props.disabled ? true : null}
```

Comment:
Is there a reason for those conditions? Like it looks like it's the same as doing; `if (boolean) {true} else {false}`

Reply:
the type is `boolean | undefined`. It looks like not ok to set `disabled = undefined`

Reply:
Well basically if you don't add the props `disabled` to `<Field />` by default it'll be undefined. So the type of `disabled` is `boolean | undefined` which is matching our `disabled` props. So imo it should work, but this is really some details, and I don't really want to bother you which such a thing as it's really quick, so I'll check myself

Reply:
Ah, I got it! Thanks!